app/uk/gov/hmrc/tradergoodsprofilesrouter/service/CreateProfileService.scala

```diff
@@ -82,8 +84,12 @@ class CreateProfileService @Inject() (
     )
 
   private def padNiphlNumber(niphlNumber: Option[String]): Option[String] =
```

Comment:
I wonder if this could be in a PadNiphlNumberPadding tread and be reused.

Reply:
Yeah I did consider this, but then you would have to make it a service so you can inject the appConfig in it, then that needs to be injected everywhere it is needed. I thought for such a small piece of functionality that will be deleted soon that it would be overkill